theme/static/js/dashboard.js

```diff
@@ -0,0 +1,74 @@
+function initializeTable() {
```

Comment:
let's consolidate js into a supporting file @Maurier hopefully agrees

Reply:
Are you proposing a overall clean up or Is this necessary for this shipment?

Reply:
In this PR

Reply:
@cuahsimarko Are you referring the method initializeTable()?

Reply:
@Maurier @cuahsimarko This js is dedicated to dashboard. And the future iterations will add more functionalities and interactions on this.